operator/pkg/central/values/translation/translation.go

```diff
@@ -70,8 +76,19 @@ func translate(c platform.Central) (chartutil.Values, error) {
 	if centralSpec == nil {
 		centralSpec = &platform.CentralComponentSpec{}
 	}
-
-	v.AddChild("central", getCentralComponentValues(centralSpec))
+	annotations := c.GetAnnotations()
+	var obsoletePvc bool
+	if value, ok := annotations[common.CentralPVCObsoletedAnnotation]; ok {
+		obsoletePvc = strings.EqualFold("true", strings.TrimSpace(value))
```

Comment:
Please extract a function for checking the value (or even existence too) of this annotation and reuse it in the PVC reconcile extension.

Reply:
Any comment on the above comment @c-du ? This would help ensure that these two identical checks do not diverge in the future.